### DISCUSSION TITLE
Tab close icon should always be visible

### DISCUSSION THREAD

**Discussion by rakleed** [rustdesk/rustdesk#3304] [Q&A] [answered]
### Description

The close tab icon should always be visible, as it's not at all obvious to a new user that it should be there.

### Impact

It will be easier for new users to understand the interface.

### Additional Context

For an example, you can see Google Chrome and Explorer in Windows 11.

**Comment by rustdesk** [comment, marked as answer]
Safari does not show  close icon and many popular apps do not show it, visual source code only show close icon of active tab.

**Reply by rustdesk** [reply]
But you can create a poll for these three choices. I am also wondering which one is most liked by people.

**Reply by rakleed** [reply]
Done: https://github.com/rustdesk/rustdesk/discussions/3363
